packages/builder/sample-project/algob.config.js

```diff
@@ -1,9 +1,20 @@
+import { mkAccounts } from "algob";
+
+let accounts = mkAccounts([{
+  addr: "KFMPC5QWM3SC54X7UWUW6OSDOIT3H3YA5UOCUAE2ABERXYSKZS5Q3X5IZY",
```

Comment:
I don't think any accounts should be hardcoded in the sample project. 
If you think there should already exist an account we can crash the tool if there are no accounts during the run.

Reply:
I was thinking about it. It helps for testing and bootstraping, and also shows an example. 
In any case we will have examples, and some less savy users can copy paste an account from example and use it.

I propose to add the following comment:
```
// NOTE: this is only an example. DON'T this account in any working environment because
// everyone can check it and use the private key (this account is visible to everyone).
```

Reply:
I also don't like the idea of having these accounts in the main config because it will end up in code.
For me it looks like a good design would be to have `algob.config.local.js` to hold this kind of data.

We can have the comment for now. We may need to create a separate file for local user config.

Tool crash will also work if the accounts will get deleted. Helpful info should be added there anyway.

Reply:
I want to make it possible for a user to have a quick start. that's why we also have a dev-net in the backlog.

Reply:
Re `....local.js` - what's the benefit of it, the normal config will be there anyway. And there is other function - which I created - to load accounts from a file. And this is what users should use to not commit config.

Reply:
Or accounts from `ENV`

Reply:
ok